### PR TITLE
sender email variabilized

### LINKED ISSUE #51: Can't change sender email in jail.conf
## Affected Puppet, Ruby, OS and module versions/distributions

- Puppet: ALL
- Ruby: ALL
- Distribution: ALL
- Module version: ALL

## What behaviour did you expect instead

I expect the sender email to be variabilized as for destemail.

### PR DESCRIPTION
**Pull request by ryayon**
#### Pull Request (PR) description
The `sender` email was not variablized as for the `destemail`.

#### This Pull Request (PR) fixes the following issues
   Fixes #51

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -32,6 +32,7 @@
   $action                   = 'action_mb',
   $bantime                  = 432000,
   $email                    = "fail2ban@${::domain}",
+  $sender                   = 'fail2ban@localhost',
```
Comment (ekohl) [inline manifests/init.pp]: I realize this was `@localhost` before, but should this be `fail2ban@${::fqdn}` by default?

Comment (ekohl) [reply manifests/init.pp]: I think that would be good.

Comment (ryayon) [reply manifests/init.pp]: Fixed

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**